### DISCUSSION TITLE
Question: would this allow enabling software locked options?

### DISCUSSION THREAD

**Discussion by coaxial** [cedricp/ddt4all#1161] [Q&A]
Hello, I'm wondering if this can help enable software locked options like the 360 degrees camera on a Megan e-tech? Thanks :)

**Comment by AndyIreland** [comment]
being a e-tech its more than likely security gateway protected OBD2 port on your car , you have to get it unlocked with a Token from Renault or read this thread , also some people have claimed they have bypassed the OBD2 socket by taking apart their dash and directly wiring to the BCM ECU or back of the easylink radio connections:
https://github.com/cedricp/ddt4all/discussions/956

**Reply by coaxial** [reply]
Thank you
